### PR TITLE
Feat signout untrust

### PR DESCRIPTION
**Pull request by Karim94**
### Linked PRs in other repos
https://github.com/PeerioTechnologies/peerio-icebear/pull/91
### Relevant issue tracker links
https://app.clubhouse.io/peerio/story/358/improve-signout-to-support-latest-sdk-and-untrust
### Manual testing instructions
Verify that signout still works correctly.
Verify that parameter passed to SDK through `User.current.signout(popupResult.checked);` sends the correct value of the checkbox (true for checked, false for unchecked).
### Breaking changes
N / A
### Additional notes

### CODE REVIEW

File: app/components/login/login-state.js [hunk 1]
```diff
@@ -151,7 +151,8 @@ class LoginState extends RoutedState {
     async signOut(force) {
         const inProgress = !!fileStore.files.filter(f => f.downloading || f.uploading).length;
         await !force && inProgress ? rnAlertYesNo(tx('dialog_confirmLogOutDuringTransfer')) : Promise.resolve(true);
-        if (!force && User.current.autologinEnabled && !await popupSignOutAutologin()) {
+        const popupResult = await popupSignOutAutologin();
+        if (!force && User.current.autologinEnabled && !popupResult) {
```
Comment (seavan) [inline app/components/login/login-state.js]: this really changes the logic.

what was before:
if user didn't have autoLoginEnabled or if it was a forced signout, we wouldn't show popup and wait for it.

now - we show popup and wait for it anyway.

this is breaking the flow so it has to be fixed.

**Comment by Karim94** [comment]
@seavan fixed, but not sure if it can be done in a better way.

**Comment by Karim94** [comment]
@seavan Ready.

**Comment by seavan** [comment]
@ebilg ready for testing

**Comment by ebilg** [comment]
working on staging and labs

**Pull request merged**